easybuild/easyconfigs/s/STAR-Fusion/STAR-Fusion-1.5.0-foss-2018b-Perl-5.28.0.eb

```diff
@@ -30,16 +32,41 @@ exts_defaultclass = 'PerlModule'
 exts_filter = ("perldoc -lm %(ext_name)s ", "")
 
 exts_list = [
-    ('Set::IntervalTree', '0.10', {
-        'source_tmpl': 'Set-IntervalTree-0.10.tar.gz',
-        'source_urls': ['https://cpan.metacpan.org/authors/id/B/BE/BENBOOTH/'],
-        'checksums': ['e3bd9ccf0d074b5f879eef1ed88254983697bf83d02744fce62150ee46553ebc'],
+    ('ExtUtils::CppGuess', '0.12', {
```

Comment:
@jdemeul out of curiosity: why we you need 'ExtUtils::CppGuess' here?

Reply:
@smoors Set::IntervalTree seems to require this.

When building without it:
`== installing extension Set::IntervalTree 0.12 (1/6)...
== FAILED: Installation ended unsuccessfully (build directory: /tmp/STARFusion/1.5.0/foss-2018b-Perl-5.28.0): build failed (first 300 chars): cmd " perl Makefile.PL PREFIX=/srv/sw/eb/software/STAR-Fusion/1.5.0-foss-2018b-Perl-5.28.0 " exited with exit code 2 and output:
Can't locate ExtUtils/CppGuess.pm in @INC (you may need to install the ExtUtils::CppGuess module) (@INC contains: /srv/sw/eb/software/STAR-Fusion/1.5.0-foss-2018b-Perl-5.2`

Happy to remove again if there's a better way to fix this?